### PR TITLE
feat: add ingore security check api

### PR DESCRIPTION
**Pull request by zyxxoo**

### CODE REVIEW

File: hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java [hunk 1]
```diff
@@ -126,6 +127,17 @@
             ImmutableSet.of("newSecurityException")
     );
 
+    private static final Set<String> ignoreCheck = new CopyOnWriteArraySet<>();
+
+    public static void addIgnoreCheck(String clazz) {
+        if (callFromGremlin()) {
+            throw newSecurityException(
+                  "Not allowed to add ignore check via Gremlin");
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java:135]: add ignore-checked class

File: hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java [hunk 2]
```diff
@@ -475,6 +487,10 @@ private static boolean callFromNewSecurityException() {
         return callFromMethods(NEW_SECURITY_EXCEPTION);
     }
 
+    private static boolean callFromIgnore() {
```
Comment (javeme) [inline hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java:490]: ditto

File: hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java (hunk 1 above)
Comment (zyxxoo) [reply hugegraph-core/src/main/java/com/baidu/hugegraph/security/HugeSecurityManager.java:135]: I think this method be called when inited best, if add ignore-checked, maybe it is called on other time;

**Review by javeme — APPROVED** [review]

**Comment by zyxxoo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by zhoney — APPROVED** [review]

**Pull request merged**